firmware/application/apps/ui_playlist.hpp

```diff
@@ -60,7 +60,7 @@ class PlaylistView : public View {
 
     // More header == less spectrum view.
     static constexpr ui::Dim header_height = 6 * 16;
-    static constexpr uint32_t baseband_bandwidth = 2'500'000;
+    // static constexpr uint32_t baseband_bandwidth = 2'500'000;
```

Comment:
Just remove?

Reply:
Yes I will remove it . 
I just left  it , because I thought that you added it ,  and just to show you that we are not using it 
No problem at all, let's remove it .